src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -1362,35 +1355,22 @@ dtkPlugin* AlgorithmPaintToolbox::plugin()
     return plugin;
 }
 
-void AlgorithmPaintToolbox::setOutputMetadata(const medAbstractData * inputData, medAbstractData * outputData)
-{
-    Q_ASSERT(outputData && inputData);
-
-    QStringList metaDataToCopy;
-    metaDataToCopy
-            << medMetaDataKeys::PatientName.key()
-            << medMetaDataKeys::StudyDescription.key();
-
-    foreach( const QString & key, metaDataToCopy ) {
-        outputData->setMetaData(key, inputData->metadatas(key));
-    }
-
-    QString seriesDesc;
-    seriesDesc = tr("Segmented from ") + medMetaDataKeys::SeriesDescription.getFirstValue( inputData );
-
-    medMetaDataKeys::SeriesDescription.set(outputData,seriesDesc);
-}
-
 // new methods from music's version
 
 void AlgorithmPaintToolbox::setParameter(int channel, int value) // LOIC's method for pipeline I guess
 {
     seedPlanted = false; //prevent from crash when data not set
     if(channel == 0)
+    {
         m_wandLowerThresholdSlider->setValue(value);
-
+        m_wandLowerThresholdSpinBox->setValue(value);
```

Comment:
Aren't these two widgets actually linked ? I mean modifying one automatically changes the other